test/ValidationFramework/CrossValidator.cpp

```diff
@@ -0,0 +1,125 @@
+/// Apache License 2.0
+
+#include <ValidationFramework/CrossValidator.hpp>
+
+namespace ostk
+{
+namespace astro
+{
+namespace validation
+{
+
+Array<Array<VectorXd>> CrossValidator::IngestOutputQuantities(
```

Comment:
Worth using other containers for those for perf in the future (not saying we should do this now?) but might need some improvements for longer scenarios in the future?

Reply:
hmmm what kinds of containers? you mean `CoordinateSubsets`? Cause 1) I don't think other containers are going to be different in terms of performance and 2) I think its quite clear to have an array of timesteps and an array of coordinate subsets within that array instead of putting stuff into a MatrixXd where you don't know which dimension means what. 

What do you think?

Reply:
Yeah i was thinking MatrixXd but that might be premature optimization